pelutils/ds/plots.py

```diff
@@ -218,19 +219,20 @@ def __init__(
     def __enter__(self):
         if self._style:
             plt.style.use(self._style)
-        mpl.rcParams.update(self._rc_params)
+
+        self._rc_context = plt.rc_context(self._rc_params)
+        self._rc_context.__enter__()
 
         self.fig, self.ax = plt.subplots(self._nrow, self._ncol)
-        self.fig: mpl.figure.Figure
-        self.ax: mpl.axes.Axes
+
         return self
 
-    def __exit__(self, *_):
+    def __exit__(self, *args):
         if self._tight_layout:
             plt.tight_layout()
```

Comment:
Burde de her linjer med tight_layout og savefig ikke være på self.figure i stedet for plt?

Reply:
Jo, jeg ser lige på det. Jeg ved aldrig, hvornår man skal bruge plt og fig